zebra-network/src/peer/priority.rs

```diff
@@ -123,9 +123,11 @@ fn address_is_valid_for_outbound_connections(
             return Err(
                 "invalid peer port: port is for Regtest, but Zebra does not support that network",
             );
-        } else if [16125, 26125].contains(&peer_addr.port()) {
-            // 16125/26125 is used by Flux/ZelCash, which uses the same network magic numbers as Zcash,
-            // so we have to reject it by port
+        } else if [8033, 18033, 16125, 26125].contains(&peer_addr.port()) {
+            // These coins use the same network magic numbers as Zcash, so we have to reject them by port:
+            // - ZClassic: 8033/18033
+            //   https://github.com/ZclassicCommunity/zclassic/blob/504362bbf72400f51acdba519e12707da44138c2/src/chainparams.cpp#L130
+            // - Flux/ZelCash: 16125/26125
```

Comment:
Does Zebra also error/warn if it is configured to listen on these ports?

Reply:
Thanks for the reminder, I updated this PR to make that happen:
https://github.com/ZcashFoundation/zebra/pull/6567/files#diff-890f028f155b45da59a5072f406a55b97d98ea27b27b40bad3d716627997e318R471-R476

I think we always meant to re-use that code for the listener, we just hadn't got around to it.